app/models/server/models/OutOfOfficeRooms.ts

```diff
@@ -0,0 +1,43 @@
+import { Base } from './_Base';
+
+class OutOfOfficeRooms extends Base {
+	constructor() {
+		super('OutOfOfficeRooms');
```

Comment:
since this is the name of the collection, please use a more standard one
```suggestion
		super('outofoffice_rooms');
```

Reply:
changed the name to `outofoffice_rooms`